pyscriptjs/Makefile

```diff
@@ -64,6 +64,13 @@ test:
 	npm run build
 	$(conda_run) pytest -vv $(ARGS) tests/ --log-cli-level=warning
 
+all:
+	make examples
+	npm install
+	npm run build
+	npm run dev
```

Comment:
I think it's better to change to `make dev` just in case we change cmds in the future...